engines/bastion_katello/app/assets/javascripts/bastion_katello/ostree-branches/details/ostree-branches-details.controller.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * @ngdoc object
+ * @name  Bastion.ostree-branches.controller:OstreeBranchesDetailsController
+ *
+ * @requires $scope
+ * @requires OstreeBranch
+ *
+ * @description
+ *   Provides the functionality for the Ostree Branch action pane.
+ */
+angular.module('Bastion.ostree-branches').controller('OstreeBranchesDetailsController', ['$scope', 'OstreeBranch',
+    function ($scope, OstreeBranch) {
+        if ($scope.branch) {
+            $scope.panel = {loading: false};
+        } else {
+            $scope.panel = {loading: true};
+        }
```

Comment:
There is a new pattern for this, see https://github.com/Katello/katello/blob/master/engines/bastion_katello/app/assets/javascripts/bastion_katello/activation-keys/details/activation-key-details.controller.js#L24-L31